changes/1730.doc.rst

```diff
@@ -0,0 +1 @@
+Corrected some docstrings of WebView
```

Comment:
As noted in the docs, change notes are written as they will appear in release notes. They.
```suggestion
Docstrings for WebView have been corrected.
```

In addition, this is a very minor change; I wouldn't expect to see a release note for a handful of docstring typos; this file should be 1730.misc.rst.

Reply:
If you mean [this section](https://briefcase.readthedocs.io/en/stable/how-to/contribute-code.html#add-change-information-for-release-notes) of the docs, that actually encourages the "Corrected X" style rather than "X has been corrected". Perhaps it should be rephrased to something like "written in complete declarative sentences", which would also allow things like "X is now Y".

Reply:
That's the change note section in briefcase; the corresponding version in Toga is [this section](https://toga.readthedocs.io/en/latest/how-to/contribute-code.html#add-change-information-for-release-notes) which is framed differently. The briefcase version should definitely be updated; I'm definitely open to improvements on the Toga version as well.